appserver/pom.xml

```diff
@@ -128,9 +128,13 @@
         <javax.batch-api.version>1.0.1</javax.batch-api.version>
         <com.ibm.jbatch.container.version>1.0.2</com.ibm.jbatch.container.version>
         <com.ibm.jbatch.spi.version>1.0.2</com.ibm.jbatch.spi.version>
-        <javax.management.j2ee-api.version>1.1.2</javax.management.j2ee-api.version>
+    	<javax.management.j2ee-api.version>1.1.2</javax.management.j2ee-api.version>
 	<jboss.classfilewriter.version>1.2.1.Final</jboss.classfilewriter.version>
         <easymock.version>3.5</easymock.version>
+	<javax.resource-api.version>1.7.1</javax.resource-api.version>
+	<javax.xml.registry-api.version>1.0.8</javax.xml.registry-api.version>
+	<maven-rar-plugin.version>2.4</maven-rar-plugin.version>
```

Comment:
where is this property used ? I don't see a plugin definition for maven-rar-plugin.

Reply:
maven-rar-plugin is defined in appserver/jaxr-ra/jaxr-ra-rar/pom.xml.

Reply:
Can you instead add pluginManagement in appserver/pom.xml and omit the version under appserver/jaxr-ra/jaxr-ra-rar/pom.xml.